src/domain.ts

```diff
@@ -120,6 +174,7 @@ interface Configuration {
   theme: string | null | undefined;
   fontFamily: string | null | undefined;
   previewMarkdownFiles: boolean;
+  slidesFolder: string | "" | undefined;
```

Comment:
```suggestion
  slidesFolder: string | undefined;
```

Since it would be redundant with the `string` type ;)